frontends/election-manager/src/utils/election.ts

```diff
@@ -264,3 +264,45 @@ export function generateBlankBallots({
 
   return ballots;
 }
+
+export function generateOvervoteBallot({
+  election,
+  precinctId,
+}: {
+  election: Election;
+  precinctId: PrecinctId;
+}): Dictionary<string | VotesDict> | undefined {
```

Comment:
Worth adding a comment above the function noting when we expect to return `undefined` (I believe when there's no contest to validly overvote)